src/exports/generated/CBL.exp

```diff
@@ -30,7 +30,6 @@ _CBLLog_Callback
 _CBLLog_SetCallback
 _CBLLog_ConsoleLevel
 _CBLLog_ConsoleLevelOfDomain
-_CBLLog_SetConsoleLevel
```

Comment:
This should not be removed.

Reply:
At some point it wasn't enough to add the definition in the .def file, but I see that now, when I statically linked it's working. It might be the case that when linking the dll, _CBLLog_* should be removed from the .exp file, however that's not currently an issue.